tests/data/test_model.py

```diff
@@ -0,0 +1,86 @@
+"""Test for ModelData class"""
+
+from pathlib import Path
+import shutil
+
+from aiida_mlip.data.model import ModelData
+
+
+def test_local_file():
+    """Testing that the local file function works"""
+    # Construct a ModelData instance with the local file
+    model = ModelData.local_file("./tests/input_files/model_local_file.txt")
+    # Assert the ModelData contains the content we expect
+    content = model.get_content()
+    with open("./tests/input_files/model_local_file.txt", encoding="utf-8") as f:
+        assert content == f.read()
```

Comment:
If this were a `Path`, can use [`read_text`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text)
```suggestion
    assert content == MODEL_FILE.read_text()
```